apps/meteor/client/views/marketplace/AppMenu.js

```diff
@@ -334,20 +341,19 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 						action: handleDisable,
 					},
 				}),
-			...(app.installed &&
-				isAdminUser &&
-				!isAppEnabled && {
-					enable: {
-						label: (
-							<>
-								<Icon name='check' size='x16' marginInlineEnd='x4' />
-								{t('Enable')}
-							</>
-						),
-						disabled: !app.migrated && !appCountQuery?.data?.hasUnlimitedApps && appCountQuery?.data?.enabled >= appCountQuery?.data?.limit,
-						action: handleEnable,
-					},
-				}),
+			...(shouldBeAbleToEnable && {
```

Comment:
I think is better to use `is[something]` to name boolean variables... `isEnableable`

Reply:
I don´t think this is a necessary rule-of-thumb when naming booleans. `isSomething`, `canDoSomething`, `hasSomething` and I guess even `shouldSomething` are valid names that clearly send the message they are booleans IMO